images/manageiq-orchestrator/container-assets/entrypoint

```diff
@@ -16,6 +16,54 @@ function check_svc_status() {
   echo "${SVC_NAME}:${SVC_PORT} - accepting connections"
 }
 
+function check_deployment_status() {
+  echo "== Checking deployment status =="
+
+  cd ${APP_ROOT}
+  while true; do
+    bin/rake evm:deployment_status
+    case $? in
+      3) # new_deployment
+        echo "== Starting New Deployment =="
+
+        pushd ${APP_ROOT}
+          set -e
+          REGION=${DATABASE_REGION} bin/rake db:migrate
+          REGION=${DATABASE_REGION} bin/rake db:seed
+          create_initial_group
+        popd
+
+        break
+      ;;
+      4) # new_replica
+        echo "New replica is not supported, exiting.."
+        return 1
+      ;;
+      5) # redeployment
+        echo "== Starting Re-deployment =="
+
+        break
+      ;;
+      6) # upgrade
+        echo "== Starting Upgrade =="
+        pushd ${APP_ROOT}
+          bin/rake db:migrate
+          [ "$?" -ne "0" ] && echo "ERROR: Failed to migrate database" && return 1
+        popd
+
+        break
```

Comment:
does this break the loop or the case?  breaks are so confusing to me 😆

Reply:
Ha yeah this will break the loop, that's `C` where you need the break so it doesn't skip to the next case

Reply:
bash switch/case syntax always confuses me, the `;;` and `esac` :scream:

Reply:
We could `return 0` from there though since we're doing `return 1` for the errors, might be more obvious

Reply:
yeah return 0 might be much clearer